kube/services/jobs/indexd-authz-job.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: batch/v1
```

Comment:
talked about this in person but want to note here: we should add arguments for passing through start did and whether or not to use sheepdog db for checking program vs. project